pnpm-lock.yaml

```diff
@@ -470,37 +468,31 @@ packages:
     resolution: {integrity: sha512-D/vdxpvrtkfYITHpiQjlvhZBNkm2GLmm/lZHbSs5R+nghbulTgXR9I8pTALFTBhjHZQu55DTiACmn/irhhotkA==}
     cpu: [x64]
     os: [linux]
-    libc: glibc
```

Comment:
These I think are also not correct... Thoughts @prabhu?

Reply:
yes, the `libc` attributes are getting added with a [hooks](https://github.com/CycloneDX/cdxgen/blob/master/.pnpmfile.cjs) to workaround a pnpm bug/feature. Not sure what command renovate is invoking to update the lock file. We can do the updates manually for now till figure out the right setting.

Reply:
I'll give it a try. Maybe we can have renovate trigger a command after updating npm dependencies? @setchy is something like that possible?